tests/test-timber-twig-date-function.php

```diff
@@ -1,141 +0,0 @@
-<?php
```

Comment:
I specifically added these tests in https://github.com/timber/timber/pull/2135 to make sure that the `|date` filter in Twig works

- in the way that Twig is expecting it to 
- with WordPress dates and times

There are some custom filters involved to make sure the correct WordPress time settings are used. It doesn’t test whether Twig’s date functions work, but **whether our custom filtering doesn’t break Twig’s basic date filter**. That’s why I would advocate to keep these.

Reply:
I see, I restored the file.